schema/GenomeAnnotationEnum.mcf

```diff
@@ -997,3 +997,162 @@ name: "Estonian Genome Project"
+
+Node: dcid:GenomeAssemblyReleaseTypePatch
+name: "GenomeAssemblyReleaseTypePatch"
+typeOf: dcs:"GenomeAssemblyReleaseTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/"
+description: "The only change from the previous assembly version is the addition or modification of a patch assembly-unit (relevant for assemblies maintained by the Genome Reference Consortium )"
+
+Node: dcid:GenomeAssemblyLevelEnum
+name: "GenomeAssemblyLevelEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "The highest level of assembly for any object in the assembly."
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+
+Node: dcid:GenomeAssemblyLevelComplete_genome
+name: "GenomeAssemblyReleaseTypeComplete_genome"
+typeOf: dcs:"GenomeAssemblyLevelEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+description: "All chromosomes are gapless and have no runs of 10 or more ambiguous bases (Ns), there are no unplaced or unlocalized scaffolds, and all the expected chromosomes are present (i.e. the assembly is not noted as having partial genome representation). Plasmids and organelles may or may not be included in the assembly but if present then the sequences are gapless."
+
+Node: dcid:GenomeAssemblyLevelChromosome
+name: "GenomeAssemblyReleaseTypeChromosome"
+typeOf: dcs:"GenomeAssemblyLevelEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+description: "There is sequence for one or more chromosomes. This could be a completely sequenced chromosome without gaps or a chromosome containing scaffolds or contigs with gaps between them. There may also be unplaced or unlocalized scaffolds."
+
+Node: dcid:GenomeAssemblyLevelScaffold
+name: "GenomeAssemblyReleaseTypeScaffold"
+typeOf: dcs:"GenomeAssemblyLevelEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+description: "Some sequence contigs have been connected across gaps to create scaffolds, but the scaffolds are all unplaced or unlocalized."
+
+Node: dcid:GenomeAssemblyLevelContig
+name: "GenomeAssemblyReleaseTypeContig"
+typeOf: dcs:"GenomeAssemblyLevelEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/#level"
+description: "Nothing is assembled beyond the level of sequence contigs."
+
+Node: dcid:RefSeqCategoryEnum
+name: "RefSeqCategoryEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Denotes that the assembly is a reference or representative genome in the NCBI Reference Sequence ( RefSeq ) project classification."
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help/"
+
+Node: dcid:RefSeqCategoryReferenceGenome
+name: "RefSeqCategoryReference_genome"
```

Comment:
here and below, use consistent naming as dcid?

Reply:
converted everything to camel case